mk/util.mk

```diff
@@ -19,4 +19,6 @@ endif
 print-%:
 	@echo $*=$($*)
 
+export RUST_BACKTRACE=1
```

Comment:
I think we may want to omit this for now, this causes all tests to emit backtraces which may not always be desirable (especially the test harness itself). The change below looks good to me though!